docs/source/pages/howToWorkWithPlatform/mainApplication/templateList/templateManagement/documentManagementOnTemplate/structuredDocument/fields/table/table.rst

```diff
@@ -22,44 +21,49 @@ Create Table Field
 
 6. If you want to add column with row numbers enable show column number attribute
 
-7. Fill the quantity of the columns of the table
+7. Fill the quantity of the columns of the table (up to 120)
 
 8. Click save
 
 Fill the Table with fields
 ==========================
 
-1. Add table to the document.
+1. Add table to the document
 
-2. Drag and drop needed fields to the table cells, set and save field properties. Next fields can be added to the Table: Text, Number, Currency, Date and time, E-ink signature, Dictionary, Lookup and Duplicate fields. Adding new table inside the table impossible.
+2. Drag and drop needed fields to the table cells, set and save field properties. Next fields can be added to the Table: Text, Number, Currency, Date and time, E-ink signature, Dictionary, Lookup and Duplicate fields. Adding new table inside other table is not supported
 
 Edit Table Field properties
 ===========================
 
-1. Navigate to /template/form/create.
+1. Navigate to /template/form/create
 
 2. Click on the table field in the document
 
 3. Edit field form should appear on the right side
 
-4. Click on "Delete" button will delete field and close edit form
+4. Click on "Delete" button will delete table and close edit form
 
-5. Click on Cancel will close edit form
+5. Click on "Cancel" will close edit form
 
 6. You can change name of the field and role to whom table is assigned
 
-7. Click on "Save" will Save all the changes you mad and close edit form
+7. Click on "Save" will save all the changes you made and close edit form
+
+.. note::
+   If you want to truncate some columns of the table in it's initial state you can click on according column, enable "Initially hidden" attribute in column properties and click "Save". Note that this is supported only for columns which have optional dynamic fields in them. Columns with static text, non-optional fields or empty cells can not be hidden. On the envelope view user will be able to show such columns from the menu on the right end of the table (only columns which have optional fields assigned for viewing user will be presented). If hidden column fields will not be filled during document processing these columns will be excluded from the final version of the document.
```

Comment:
"If you want to truncate some columns" it's  better to replace with "If you want to hide some columns"